api/v1/views/states.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+"""Create a new view for State objects that handles all default RestFul API
+"""
+from flask import Flask, json, jsonify, abort, request
+from api.v1.views import app_views
+from models import storage
+from models.state import State
+
+
+@app_views.route('/states', methods=['GET'], strict_slashes=False)
+def get_states():
+    """get all states
+    """
+    tmp_list = []
+    for key, value in storage.all("State").items():
+        tmp_list.append(value.to_dict())
+    return json.dumps(tmp_list)
+
+
+@app_views.route('/states/<state_id>', methods=['GET'], strict_slashes=False)
+def get_state_by_id(state_id):
+    """get state by id
+    """
+    for key, value in storage.all("State").items():
+        if state_id == value.id:
```

Comment:
if you know the id and the class, you can acces directly to the object trought storage.get(cls,id)